src/exporter/DaemonMetricCollector.cc

```diff
@@ -335,13 +335,13 @@ DaemonMetricCollector::add_fixed_name_metrics(std::string metric_name) {
   labels_t labels;
   new_metric_name = metric_name;
 
-  std::regex re("^data_sync_from_(.*)\\.");
-    std::smatch match;
-    if (std::regex_search(metric_name, match, re) == true) {
-      new_metric_name = std::regex_replace(metric_name, re, "from_([^.]*)', 'from_zone");
+  std::regex re("data_sync_from_([^_]*)");
```

Comment:
Why now the metric does not start by "data_sync_from"? I am asking because maybe behind this "innocent" change we have other side-effects and implications.
This happens in all Ceph versions?

Reply:
This only happend to Ceph version >= reef & I've added a description in the tracker now, PTAL https://tracker.ceph.com/issues/65091?next_issue_id=65080&prev_issue_id=65110#note-1